db/models/ally.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Ally = db.define('allies', {
+  username: Sequelize.STRING,
+  phone_number: {
```

Comment:
I was thinking it would be meaningful to require a phone number of the advocates so that they can be available for our clients. That way a woman who comes to us asking for help always has a personal phone number they can call. That was the intention, what do you think?